utils.groovy

```diff
@@ -516,26 +516,32 @@ def build_artifacts(pipecfg, stream, basearch) {
 
 def get_registry_repos(pipecfg, stream, version) {
     def registry_repos = pipecfg.registry_repos ?: [:]
-    // merge top-level registry_repos with stream-specific bits
-    registry_repos += pipecfg.streams[stream].additional_registry_repos ?: [:]
+    // get stream-specific bits additional repos
+    def additional_repos = pipecfg.streams[stream].additional_registry_repos ?: [:]
```

Comment:
additional_registry_repos isn't added in the docs section of this PR

Reply:
Yeah it's not something I introduced there, but it's indeed not documented.
I'll add it